### PR TITLE
fix(tests): use current package in e2e tests

### LINKED ISSUE #194: e2e workflow spec should use a linked angular cli
Currently the e2e tests are doing an npm install to grab the version of angular-cli to use for test execution, it should be changed to use npm link to utilize the local version of itself to test functionality.

The line in question is:
sh.exec('npm i angular-cli -g -C ' + process.cwd(), { silent: true });

### PR DESCRIPTION
**Pull request by filipesilva**
Close #194

### CODE REVIEW

File: tests/e2e/e2e_workflow.spec.js [hunk 1]
```diff
@@ -18,11 +18,10 @@ describe('Basic end-to-end Workflow', function () {
   it('Installs angular-cli correctly', function() {
     this.timeout(300000);
 
+    sh.exec('npm link', { silent: true });
```
Comment (hansl) [inline tests/e2e/e2e_workflow.spec.js]: Do you need to pass in the path to angular-cli?

**Comment by hansl** [comment]
Thanks!  I was about to spend time on this, but if this works then :+1:  Lets see what Travis says.

File: tests/e2e/e2e_workflow.spec.js (hunk 1 above)
Comment (filipesilva) [inline tests/e2e/e2e_workflow.spec.js]: If I got this correctly, tests are always initiated from the root, which is why the setup with `tmp` dir works.

Although to be perfectly honest I couldn't find what the `-C` flag in `npm install` did in the command I removed.

Comment (hansl) [inline tests/e2e/e2e_workflow.spec.js]: Yeah I'm pretty sure you need to pass in the path to angular-cli.

**Comment by filipesilva** [comment]
There's something missing actually, also need to link *inside* the generated project. sec

File: tests/e2e/e2e_workflow.spec.js (hunk 1 above)
Comment (filipesilva) [inline tests/e2e/e2e_workflow.spec.js]: [npm link](https://docs.npmjs.com/cli/link) doesn't seem to take a path to link globally, only to link locally. Is there a missing argument in these docs that you know of?

File: tests/e2e/e2e_workflow.spec.js [hunk 2]
```diff
@@ -18,11 +18,10 @@ describe('Basic end-to-end Workflow', function () {
...
     return tmp.setup('./tmp')
       .then(function () {
         process.chdir('./tmp');
-
-        sh.exec('npm i angular-cli -g -C ' + process.cwd(), { silent: true });
```
Comment (hansl) [inline tests/e2e/e2e_workflow.spec.js]: do `npm link` from here?

**Comment by filipesilva** [comment]
Closing this PR for now, as it it not fixing what is it intended to fix.

**Pull request closed without merging**